contracts/index_executor/src/actions/polkadot/mod.rs

```diff
@@ -0,0 +1,48 @@
+mod xcm_v2;
+mod xcm_v3;
+
+use crate::actions::base::sub_transactor;
+use crate::chain::Chain;
+use crate::constants::{ASTAR_PARACHAIN_ID, MOONBEAM_PARACHAIN_ID, PHALA_PARACHAIN_ID};
+use crate::{account::AccountType, call::CallBuilder};
+use alloc::{boxed::Box, string::String, vec, vec::Vec};
+
+pub fn create_actions(_chain: &Chain) -> Vec<(String, Box<dyn CallBuilder>)> {
+    vec![
+        (
+            String::from("polkadot_bridge_to_phala"),
+            Box::new(xcm_v3::PolkadotXcm::new(
+                PHALA_PARACHAIN_ID,
+                AccountType::Account32,
+                false,
+            )),
+        ),
+        (
+            String::from("polkadot_bridge_to_moonbeam"),
+            Box::new(xcm_v2::PolkadotXcm::new(
+                MOONBEAM_PARACHAIN_ID,
+                AccountType::Account20,
+            )),
+        ),
+        (
+            String::from("polkadot_bridge_to_astar_evm"),
+            Box::new(xcm_v3::PolkadotXcm::new(
+                ASTAR_PARACHAIN_ID,
+                AccountType::Account32,
+                true,
```

Comment:
And does this mean Astar only support XCM V3 message to be send directly to their EVM?

Reply:
Similar to moonbeam, the parameters of this transaction were obtained from the astar official app, so I cannot be absolutely certain.